### PR TITLE
fix: iOS 13 deprecation warnings for UI key command and setMenuVisible  IC-65, IC-70

### PR DESCRIPTION
**Pull request by agisilaos**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/IC-65" title="IC-65" target="_blank">SubtaskIC-65</a>  'init(input:modifierFlags:action:discoverabilityTitle:)' was deprecated in iOS 13.0
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we fix deprecation warnings for UIKeyCommand and setMenuVisible.

- 'setMenuVisible(_:animated:)' was deprecated in iOS 13.0: Use showMenuFromView:rect: or hideMenuFromView: instead.
- 'init(input:modifierFlags:action:discoverabilityTitle:)' was deprecated in iOS 13.0

Fixes: IC-70

----

##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by KaterinaWire — APPROVED** [review]

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift [hunk 1]
```diff
@@ -27,48 +27,30 @@ extension ConversationInputBarViewController {
...
             )
         }
 
         if inputBar.isEditing {
             commands.append(
-                UIKeyCommand(input: UIKeyCommand.inputEscape,
-                             modifierFlags: [],
-                             action: #selector(escapePressed),
-                             discoverabilityTitle: Shortcut.cancelEditingMessage)
+                UIKeyCommand(action: #selector(escapePressed), input: UIKeyCommand.inputEscape, modifierFlags: [], discoverabilityTitle: Shortcut.cancelEditingMessage)
```
Comment (ghuMustafa) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift:41]: Same as above (arguments alignment).

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift [hunk 2]
```diff
@@ -27,48 +27,30 @@ extension ConversationInputBarViewController {
...
             )
         } else if inputBar.textView.text.isEmpty {
             commands.append(
-                UIKeyCommand(input: UIKeyCommand.inputUpArrow,
-                             modifierFlags: [],
-                             action: #selector(upArrowPressed),
-                             discoverabilityTitle: Shortcut.editLastMessage)
+                UIKeyCommand(action: #selector(upArrowPressed), input: UIKeyCommand.inputUpArrow, modifierFlags: [], discoverabilityTitle: Shortcut.editLastMessage)
```
Comment (ghuMustafa) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift:45]: Same as above (arguments alignment).

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift [hunk 3]
```diff
@@ -27,48 +27,30 @@ extension ConversationInputBarViewController {
...
             )
         } else if let mentionsView = mentionsView as? UIViewController, !mentionsView.view.isHidden {
             commands.append(
-                UIKeyCommand(input: UIKeyCommand.inputUpArrow,
-                             modifierFlags: [],
-                             action: #selector(upArrowPressedForMention),
-                             discoverabilityTitle: Shortcut.choosePreviousMention)
+                UIKeyCommand(action: #selector(upArrowPressedForMention), input: UIKeyCommand.inputUpArrow, modifierFlags: [], discoverabilityTitle: Shortcut.choosePreviousMention)
```
Comment (ghuMustafa) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift:49]: Same as above (arguments alignment).

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift [hunk 4]
```diff
@@ -27,48 +27,30 @@ extension ConversationInputBarViewController {
...
             )
 
             commands.append(
-                UIKeyCommand(input: UIKeyCommand.inputDownArrow,
-                             modifierFlags: [],
-                             action: #selector(downArrowPressedForMention),
-                             discoverabilityTitle: Shortcut.chooseNextMention)
+                UIKeyCommand(action: #selector(downArrowPressedForMention), input: UIKeyCommand.inputDownArrow, modifierFlags: [], discoverabilityTitle: Shortcut.chooseNextMention)
```
Comment (ghuMustafa) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift:53]: Same as above (arguments alignment).

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift (hunk 1 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift]: Done!

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift [hunk 5]
```diff
@@ -27,48 +27,30 @@ extension ConversationInputBarViewController {
...
             commands.append(
-                UIKeyCommand(input: "\r",
-                             modifierFlags: .shift,
-                             action: #selector(shiftReturnPressed),
-                             discoverabilityTitle: Shortcut.newline)
+                UIKeyCommand(action: #selector(shiftReturnPressed), input: "\r", modifierFlags: .shift, discoverabilityTitle: Shortcut.newline)
             )
         }
 
         if inputBar.isEditing {
```
[the remark this answers is not in the document]
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift:45]: Done!

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift (hunk 3 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift]: Done!

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift (hunk 2 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift]: Done!

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift (hunk 4 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+KeyCommand.swift]: Done!

**Pull request merged**